locales/en/plugin__kubevirt-plugin.json

```diff
@@ -539,6 +539,7 @@
   "Invalid certificate, please visit the following URL and approve it": "Invalid certificate, please visit the following URL and approve it",
   "Invalid MAC address format": "Invalid MAC address format",
   "Invalid URL": "Invalid URL",
+  "Invalid YAML": "Invalid YAML",
```

Comment:
@apinnick PTAL about this. Do you agree on adding this error message for invalid YAML? Thanks.